### PR TITLE
shell: Remove unused #server css rules

### PR DESCRIPTION
**Pull request by marusak**
Noticed while working on the new navigation.
If someone can point me where this is being used I'll be happy, but I could not find a single place.

### CODE REVIEW

File: pkg/shell/shell.scss [hunk 1]
```diff
@@ -353,15 +353,6 @@ thead.credential-lock.credential-unlocked > *:last-child {
     display: table-row !important;
 }
 
-
-/* The server avatar */
-#server-avatar {
```
Comment (martinpitt) [inline pkg/shell/shell.scss:358]: This indeed seems obsolete. We use `#machine-avatar` and various other *-avatar in shell and dashboard, but not this one.

File: pkg/shell/shell.scss [hunk 2]
```diff
@@ -438,62 +429,6 @@ thead.credential-lock.credential-unlocked > *:last-child {
...
-    }
-
-    /* server overview dashboard */
-    #server .info-table-ct-container td button {
-        max-width: 100%;
-    }
-}
-
-@media (max-width: 991px) {
-    #server .info-table-ct-container {
```
Comment (martinpitt) [inline pkg/shell/shell.scss:468]: We could also drop it from pkg/lib/table.css then?

Comment (marusak) [reply pkg/shell/shell.scss:468]: Not sure what you mean. I removed '#server', not `.info-table-ct-container`. (Maybe that is also obsolete? Haven't checked)

Comment (martinpitt) [reply pkg/shell/shell.scss:468]: Yes, I don't  see any use of .info-table-ct-container anywhere.

**Pull request merged**